eng/tools/GenerateFiles/GenerateFiles.csproj

```diff
@@ -15,10 +17,14 @@
       <_TemplateProperties>
         AspNetCorePatchVersion=$(AspNetCorePatchVersion);
         DefaultNetCoreTargetFramework=$(DefaultNetCoreTargetFramework);
+        DotnetDumpVersion=$(DotnetDumpVersion);
+        dotnetefVersion=$(dotnetefVersion);
```

Comment:
minor nit: casing of dotnetefVersion here is wonky compared to everything else like DotnetDump

Reply:
I agree but it has to match the casing of the dependency in Version.Details.xml for strange reasons related to the wild logic in the https://github.com/dotnet/aspnetcore/blob/main/eng/Dependencies.props#L218 item group.

I may see if property name case-insensitivity makes a difference these days but will leave that for another PR (even though this one is failing validation).